adm/src/commands/genesis.rs

```diff
@@ -124,3 +126,51 @@ fn validate_depedencies(batches: &[Batch]) -> Result<(), CliError> {
     }
     Ok(())
 }
+
+fn check_required_settings(batches: &[Batch]) -> Result<(), CliError> {
```

Comment:
The python cli has some tests for this.  Any that can be added here?

Reply:
Added a unit test for the updated functionality